tasks.py

```diff
@@ -0,0 +1,426 @@
+from invoke import task, run 
+import platform 
+import os 
+
+PLATFORM=platform.system()
+MESSAGE_FAILED="failed to install. Please, use \"invoke task_name --verbose\" to check out the stderr and stdout responses."
+MESSAGE_OK="successfully installed"
+MESSAGE_WRONG_PLATFORM="Unsupported Platform. Only Ubuntu (16.04+), Debian (9+), Mac OS. Your system is {}".format(platform.platform())
+MESSAGE_ALREADY_INSTALLED="already installed"
+SERVE_CELERY="celery worker --app=project --loglevel=INFO -Q builds,email,cleanup"
+SERVE_HTTPS_SERVER="python manage.py runsslserver --certificate certs/local_lookit.mit.edu.pem --key certs/local_lookit.mit.edu-key.pem"
+SERVE_HTTP_SERVER="python manage.py runserver"
+SERVE_NGROK="ngrok http https://127.0.0.1:8000"
+BASE_DIR=os.getcwd()
+PATH_TO_CERTS=os.path.join(BASE_DIR, 'certs')
+
+
+@task
+def system_setup(c,verbose=False):
+    """
```

Comment:
Running `black` should fix your doc formatting issues, but in case it doesn't: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Your docstrings should look like that

Reply:
^^^ make sure to do that everywhere, actually

Reply:
Gotcha!